src/stream/src/executor/source/fs_fetch_executor.rs

```diff
@@ -413,6 +414,7 @@ impl<S: StateStore, Src: OpendalSource> FsFetchExecutor<S, Src> {
                                 yield Message::Chunk(chunk);
                             }
                             None => {
+                                tracing::debug!("Deleting file: {:?}", reading_file);
```

Comment:
Keep per file log to debug level.

Reply:
do you think we should add one metric here as it is an event show the consumption at file level.